specs/observable-cacheable.decorator.spec.ts

```diff
@@ -771,6 +771,97 @@ strategies.forEach(s => {
        * the service call count won't be incremented
        */
       expect(mockServiceCallSpy).toHaveBeenCalledTimes(3);
+      GlobalCacheConfig.shouldCacheDecider = undefined;
+    });
+
+    it('only cache data when a specific response (set from globalConfig) is returned, otherwise it should bail to service call', () => {
+      GlobalCacheConfig.shouldCacheDecider = (response: { payload: string }) => {
```

Comment:
@angelnikolov After adding these specs which manipulate ```GlobalCacheConfig``` the specs fails randomly. Can you advise how to fix this?